src/crypto/pwhash/scryptsalsa208sha256.rs

```diff
@@ -124,6 +124,28 @@ pub fn derive_key<'a>(
     }
 }
 
+/// `derive_key_interactive()` is a shortcut function for `derive_key()` with
+/// interactive limits (i.e. using `derive_key()` with `OPSLIMIT_INTERACTIVE`
+/// and `MEMLIMIT_INTERACTIVE`)
+pub fn derive_key_interactive<'a>(
```

Comment:
Do you need that `<'a>`?

Reply:
Yes, the compiler will fail if `'a` is missing.